staging/src/k8s.io/apiserver/pkg/endpoints/installer.go

```diff
@@ -427,10 +427,8 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 		actions = appendIf(actions, action{"DELETE", itemPath, nameParams, namer, false}, isDeleter)
 		actions = appendIf(actions, action{"WATCH", "watch/" + itemPath, nameParams, namer, false}, isWatcher)
 		// We add "proxy" subresource to remove the need for the generic top level prefix proxy.
-		// The generic top level prefix proxy is deprecated in v1.2, and will be removed in 1.3, or 1.4 at the latest.
-		// TODO: DEPRECATED in v1.2.
+		// The generic top level prefix proxy is deprecated in v1.2, and should be removed in v2 API.
```

Comment:
This updated comment isn't correct.  The original is correct.  The top level proxy should be removed

Reply:
I once tried to remove it but some tests failed. Seems like something still depends on it. I will reproduce it.